docs/logs/README.md

```diff
@@ -72,6 +73,42 @@ package, regardless of the .NET runtime version being used:
 
 ## Logging API
 
+### ILogger
+
+.NET supports high performance, structured logging via the
+[`Microsoft.Extensions.Logging.ILogger`](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger)
+interface (including
+[`ILogger<TCategoryName>`](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger-1))
+to help monitor application behavior and diagnose issues.
+
+#### Get Logger
+
+In order to use the `ILogger` interface, you need to first get a logger. How to
+get a logger depends on two things:
+
+* The type of application you are building.
+* The place where you want to log.
+
+Here is the rule of thumb:
+
+* If you are building an application with [dependency injection
+  (DI)](https://learn.microsoft.com/dotnet/core/extensions/dependency-injection)
+  (e.g. [ASP.NET Core](https://learn.microsoft.com/aspnet/core) and [.NET
+  Worker](https://learn.microsoft.com/dotnet/core/extensions/workers)), in most
+  cases you should use the logger provided by DI, there are special cases when
+  you want log before DI logging pipeline is available or after DI logging
+  pipeline is disposed. Refer to the [.NET official
+  document](https://learn.microsoft.com/dotnet/core/extensions/logging#integration-with-hosts-and-dependency-injection)
+  and [Getting Started with OpenTelemetry .NET Logs in 5 Minutes - ASP.NET Core
+  Application](./getting-started-aspnetcore/README.md) tutorial to learn more.
+* If you are building an application without DI, create a
+  [LoggerFactory](#loggerfactory) instance and configure OpenTelemetry to work
```

Comment:
would be nice to call out to cache the logger instead of repeatedly asking the logger factory

Reply:
Good callout. Let me add an actual benchmark for this.